app/controllers/events_controller.rb

```diff
@@ -49,13 +51,17 @@ def create
 
   def update
     @event.update(event_params)
+    Pool.recalculate(@event)
+
     flash[:notice] = 'Event was successfully updated.'
 
     redirect_to events_path
   end
 
   def destroy
     @event.destroy
+    Pool.recalculate(@event)
```

Comment:
We need also to handle the scenario where we update the conversion rates for USD.
In such case we need to wipe the pool too.

Reply:
That is true. The whole cache invalidation thing is hard 😬